src/pages/Beta/Airdrop2/index.tsx

```diff
@@ -1,26 +1,12 @@
-import React, { useEffect } from 'react'
+import React from 'react'
 import { useParams } from 'react-router-dom'
-import { AIRDROP_MENU_LINK, MENU_LINK } from 'src/constants'
-import { useChainId } from 'src/hooks'
-import { shouldHideChildItem, switchNetwork } from 'src/utils'
+import { AIRDROP_MENU_LINK } from 'src/constants'
 import AirdropUI from '../Airdrop2/airdrop'
 import HederaAirdropUI from '../Airdrop2/hederaAirdrop'
-import { useActiveWeb3React, useWalletModalToggle } from '@pangolindex/components'
-import { CHAINS, ChainId } from '@pangolindex/sdk'
 export type AirdropProps = Record<'type', AIRDROP_MENU_LINK>
 
 const Airdrop = () => {
   const params = useParams<AirdropProps>()
-  const { account } = useActiveWeb3React()
-  const chainId = useChainId()
-  const toggleWalletModal = useWalletModalToggle()
-
-  useEffect(() => {
-    if (!account) toggleWalletModal()
-    else if (chainId && shouldHideChildItem(chainId, MENU_LINK.airdrop, params?.type as AIRDROP_MENU_LINK)) {
-      switchNetwork(CHAINS[ChainId.FUJI])
-    }
-  }, [chainId, params?.type, account])
```

Comment:
any reason to remove this change ?
there are few things going here
1. if user is not connected then it will ask to connect wallet
2. if use try to open /airdrop/hedera and if that page is not supported by connected chain then it will switch network to Fuji
let me know if those are not needed now.

Reply:
1- the airdrop already does that, so we don't need to open the modal wallet automatically 
![image](https://github.com/pangolindex/interface/assets/37405304/581cce2c-7943-4980-99de-d68fe1ae78b1)
2- switching to fuji is only required in the registration process, and that has already ended